internal/auth/oidc/repository_auth_method_read.go

```diff
@@ -187,6 +190,7 @@ type authMethodAgg struct {
 	ApiUrl                            string
 	Auds                              string
 	Certs                             string
+	Scopes                            string
```

Comment:
nit: consider renaming to ClaimScopes since there is the naming collision with boundary scopes.

Reply:
ty!